MonkeyBot/Common/MonkeyHelpers.cs

```diff
@@ -69,17 +69,23 @@ public static async Task<string> ReadTextAsync(string filePath)
         /// <param name="guildID">Id of the Discord guild</param>
         /// <param name="channelID">Id of the Discord channel</param>
         /// <param name="text">Text to post</param>
-        public static async Task<IUserMessage> SendChannelMessageAsync(IDiscordClient client, ulong guildID, ulong channelID, string text, bool isTTS = false, Embed embed = null, RequestOptions options = null)
+        public static async Task<IUserMessage> SendChannelMessageAsync(IDiscordClient client, ulong guildID, ulong channelID, string text, bool isTTS = false, Embed? embed = null, RequestOptions? options = null)
```

Comment:
client could also be null

Reply:
Yeah, I guess everything from the Discord API is nullable.